apps/studio/src/features/dashboard/components/DashboardLayout.tsx

```diff
@@ -11,6 +11,51 @@ import {
 } from "@chakra-ui/react"
 import { Breadcrumb } from "@opengovsg/design-system-react"
 
+import type { RouterOutput } from "~/utils/trpc"
+import { getFolderHref } from "~/utils/resource"
+
+/**
+ * NOTE: This returns the path from root down to the parent of the element.
+ * The element at index 0 is always the root
+ * and the last element is always the parent of the current folder
+ */
+export const getBreadcrumbsFromRoot = (
```

Comment:
shifted from `apps/studio/src/pages/sites/[siteId]/folders/[folderId]/index.tsx` to be reusable across both flows